Makefile

```diff
@@ -61,9 +61,21 @@ NULL_ENTROPY_WARN_L3=****  AND IS *NOT* SUITABLE FOR PRODUCTION USE     ****\n
 
 NULL_ENTROPY_WARNING=\n$(WARNING_BORDER)$(NULL_ENTROPY_WARN_L1)$(NULL_ENTROPY_WARN_L2)$(NULL_ENTROPY_WARN_L3)$(WARNING_BORDER)
 
+WARNING_BORDER_LONG      =****************************************************************************\n
+CTR_DRBG_128_BIT_KEY_WARN_L1=****  WARNING!  MBEDTLS_CTR_DRBG_USE_128_BIT_KEY defined!               ****\n
+CTR_DRBG_128_BIT_KEY_WARN_L2=****  Using 128-bit keys for CTR_DRBG limits the security of generated  ****\n
+CTR_DRBG_128_BIT_KEY_WARN_L3=****  keysand operations that use random values to 128-bit security     ****\n
```

Comment:
"keysand" -> "keys and"
"random values"->"generated material" or "generated values" (as @yanesca prefers)

Is there a reason this text is different from the CMake version? I like the Makefile version better, but please use whatever @yanesca recommends for this text as the wording needs to be very particular. All three warnings: the doxygen "\warning", the CMake, and the Makefile should probably all be the same exact wording.

@yanesca Please recommend the exact text we should use for the warnings.